packages/amplify-cli-core/API.md

```diff
@@ -791,6 +792,12 @@ export class HooksMeta {
 // @public (undocumented)
 export type HooksNoun = 'notifications' | 'analytics' | 'api' | 'auth' | 'function' | 'hosting' | 'interactions' | 'predictions' | 'storage' | 'xr' | 'codegen' | 'env';
 
+// @public (undocumented)
+export type HooksRuntime = {
+    runtimePath: string;
+    runtime_options?: string[];
+};
```

Comment:
The casing in this file is all over the place.
We should pick one style and stick to it.

**BTW**
This type isn't used by any function exported from cli core package. We should hide it.